server/server.js

```diff
@@ -20,27 +20,25 @@ const upload = multer({
       callback(null, path);
     },
     filename: (req, file, cb) => {
-      cb(null, file.originalname);
+      cb(null, Date.now() + '-' + file.originalname);
     }
   })
 });
 
-
 app.get("/tasks", function (req, res) {
   console.log("get tasks");
   res.send({ tasks });
 });
 
 app.post('/addTask', upload.single('files'), function (req, res, next) {
-  console.log(req.body);
   const { summary, date } = req.body;
-  console.log("req.files", req.files);
+  console.log("req.files!", req.file);
   const id = Math.floor(Math.random() * (maxId - minId)) + minId;
```

Comment:
Math.random is not good enough for IDs, you can use uuid library instead of this function